.travis.yml

```
@@ -53,11 +53,20 @@ before_install:
     fi
 install: true
 script: |
-        if [ -z "${UNENCRYPTED_PACKAGE:-}" ]; then
-        citus_package -p ${TARGET_PLATFORM} 'local' 'release'
+        if [ -z "${UNENCRYPTED_PACKAGE:-}" ] && [ ${TRAVIS_EVENT_TYPE} != 'cron' ] ; then
```

below, we check if `[ ${TRAVIS_EVENT_TYPE} == 'push' ] || [ ${TRAVIS_EVENT_TYPE} == 'pull_request' ] || [ ${TRAVIS_EVENT_TYPE} == 'api' ]` but here we check if `[ ${TRAVIS_EVENT_TYPE} != 'cron' ]`

I think here we should also do below:
```suggestion
        if [ -z "${UNENCRYPTED_PACKAGE:-}" ] && ([ ${TRAVIS_EVENT_TYPE} == 'push' ] || [ ${TRAVIS_EVENT_TYPE} == 'pull_request' ] || [ ${TRAVIS_EVENT_TYPE} == 'api' ]) ; then
```

---

or, should we simply check `if !cron` in below too ?